src/effects/VST/VSTEffect.cpp

```diff
@@ -1005,12 +1005,12 @@ namespace
 {
    struct VSTEffectMessage : EffectInstance::Message
    {
-      typedef std::vector<std::optional<double> > ParamVector;
+      using ParamVector = std::vector<std::optional<double> >;
 
       // Make a message from a chunk and ID-value pairs
-      explicit VSTEffectMessage(const std::vector<char>& chunk, const ParamVector& params)
-         : mChunk(chunk),
-           mParamsVec(params)
+      explicit VSTEffectMessage(const std::vector<char>&& chunk, const ParamVector&& params)
```

Comment:
Remove the `const` qualifiers from the rvalue reference parameters.

Rvalue reference to const is possible in C++ but it's weird.

Reply:
ok, done